contributing/workflow.rst

```diff
@@ -170,6 +170,47 @@ that are easy for others to read.
 * Avoid committing code that has been commented out. If we need it
   again, it's in the git history.
 
+Why the "size" of commits matters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We said about that you should only make one change per commit. This is for a few
```

Comment:
Well remembered!

This looks good with one nit - I assume "about" here should be "above".  I'll squash that fix and merge this.